src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        if (user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
Let's simplify it) You can choose beetwen 2 versions
1)
```suggestion
        return user.getPassword().equals(password);
```
2) 
```suggestion
        return user != null && user.getPassword().equals(password);
```